### DISCUSSION TITLE
Support setting the default view to an extra view

### DISCUSSION THREAD

**Discussion by D3SOX** [DigiLive/mushroom-strategy#139] [Ideas]
How to set the default view to one from `extra_views` aka set its order to 0?

**Comment by DigiLive** [comment]
`extra_views` are appended to the builtin views in the same order as they are defined.

**Reply by D3SOX** [reply]
Then I guess this is more of a feature request than a question

**Reply by DigiLive** [reply]
Can you be more explicit?
Do you mean you want to...

- "mix" all of the built-in views and extra views by setting an order?
- set a view to open when going to the dashboard?

**Reply by D3SOX** [reply]
I created an extra view (which is the one with the arrow pointing to it
![image](https://github.com/AalianKhan/mushroom-strategy/assets/24937357/a608fb75-de89-49f0-8ebd-97e0c2b9b438)
I have set Mushroom as my user Dashboard and I want my extra view to directly open when I open HomeAssistant and go to the Dashboard
Ideally, it should also be the first icon.

**Comment by DigiLive** [comment]
Created issue #142 to follow up this request.
